scripts/test_wheel.sh

```diff
@@ -92,9 +101,27 @@ if [[ -n "${USE_DOCKER}" ]]; then
   exit 0
 fi
 
-test -d deps/env || ./scripts/install_python_toolchain.sh
+test -d deps/env || USE_MINIMAL="$USE_MINIMAL" ./scripts/install_python_toolchain.sh
 source deps/env/bin/activate
-pip install target/turicreate-*.whl
+
+bdist_wheels=($(ls target/turicreate-*.whl))
```

Comment:
This script can be used manually too, so we should make this a bit more robust -- just error out if the exact one requested is not found.

Reply:
done.